### PR TITLE
Crew StackDesc remove random )

### PR DESCRIPTION
**Pull request by Northmoc**

### CODE REVIEW

File: forge-game/src/main/java/forge/game/card/CardFactoryUtil.java [hunk 1]
```diff
@@ -3401,7 +3401,7 @@ public void resolve() {
                     "CostDesc$ Crew " + power + " (Tap any number of creatures you control with total power " + power +
                     " or more: | Crew$ True | Secondary$ True | Defined$ Self | Types$ Creature,Artifact | " +
                     "RemoveCardTypes$ True | StackDescription$ SpellDescription | SpellDescription$ CARDNAME becomes" +
-                    " an artifact creature until end of turn.)";
+                    " an artifact creature until end of turn.";
```
Comment (Agetian) [inline forge-game/src/main/java/forge/game/card/CardFactoryUtil.java:3404]: Isn't this ) a part of the initial description - the one that opens at "...(Tap any number..."?

Comment (Northmoc) [reply forge-game/src/main/java/forge/game/card/CardFactoryUtil.java:3404]: Hmm, good point

Comment (Hanmac) [reply forge-game/src/main/java/forge/game/card/CardFactoryUtil.java:3404]: maybe Sepratate the StackDescription from the SpellDescription?
i mean the StackDescription can just be `Crew power` ?

PS: also Crew shouldn't remove CardTypes, see:

> 205.1b  ... Some effects state that an object becomes an “artifact creature”; these effects also allow the object to retain all of its prior card types and subtypes. ...

**Review by Hanmac — APPROVED** [review]
👍

**Pull request merged**